cassandane/Cassandane/Cyrus/TestCase.pm

```diff
@@ -889,6 +910,13 @@ sub tear_down
     if (defined $self->{instance})
     {
         eval { push @stop_errors, $self->{instance}->stop() };
+
+        # Clean up files outside basedir
+        my @delfiles = grep defined, (
+            $self->{instance}->{config}->get('http_jwt_key_path'),
+        );
```

Comment:
```suggestion
        my @delfiles = grep { defined } (
            $self->{instance}->{config}->get('http_jwt_key_path'),
        );
```

It's unusual for me to see the `grep EXPR, LIST` version of grep.  I think the `grep BLOCK LIST` version is generally preferred, though if there was a good reason for this I have long forgotten it!

I think if I personally were going to use the `grep EXPR, LIST` version for some reason (that I can't imagine), I would put the arguments in parens, c-style, something like `grep(defined, $self->{instance}->config}...);`, to make it more visually apparent to my future self that something "unusual" was happening.

Looks like, in Cassandane, we do use a bit of everything, but strongly prefer BLOCK:

```
ellie@debian:master:~/fastmail/cyrus-imapd$ack -ch 'grep \{' cassandane/Cassandane/
181
ellie@debian:master:~/fastmail/cyrus-imapd$ack -ch 'grep [^\{]' cassandane/Cassandane/
4
ellie@debian:master:~/fastmail/cyrus-imapd$ack -ch 'grep\(' cassandane/Cassandane/
1
```